Source/cmServerProtocol.cxx

```diff
@@ -257,7 +257,7 @@ bool cmServerProtocol::DoActivate(const cmServerRequest& /*request*/,
 
 std::pair<int, int> cmServerProtocol1::ProtocolVersion() const
 {
-  return std::make_pair(1, 2);
+  return std::make_pair(1, 3);
 }
 
 std::pair<int, int> cmServerProtocol2::ProtocolVersion() const
```

Comment:
Do we need to change the minor version for this one too? So 2.1?

Reply:
Whoops. Didn't notice this. I think this should be 2.1 now.